aries-site/src/examples/components/selector/SupportSelector.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+
+import { Box, PageContent, Text, Heading, Page, TextInput } from 'grommet';
+import { Iteration, Search as SearchIcon } from 'grommet-icons';
+import { SelectorGroup, Selector } from 'aries-core';
+
+const services = [
+  {
+    value: 'aruba',
+    title: 'Aruba Central',
+  },
+  {
+    value: 'green lake portal',
+    title: 'HPE GreenLake Portal',
+  },
+  // {
+  //   value: 'block storage',
+  //   title: 'Block Storage',
+  // },
+  // {
+  //   value: 'complete care',
+  //   title: 'HPE Complete Care',
+  // },
+];
+
+const devices = [
+  {
+    value: 'aruba',
+    title: 'Aruba AP-635 WiFi 6E',
+    description: <Text size="xlarge">22</Text>,
+    icon: <Iteration aria-hidden height="medium" />,
+  },
+  {
+    value: 'green lake portal',
+    title: 'Aruba AP-635 WiFi 6E',
+    description: <Text size="xlarge">18</Text>,
+    icon: <Iteration height="medium" />,
+  },
+  {
+    value: 'block storage',
+    title: 'Aruba AP-635 WiFi 6E',
+    description: <Text size="xlarge">26</Text>,
+    icon: <Iteration height="medium" />,
+  },
+  {
+    value: 'complete care',
+    title: 'Aruba AP-635 WiFi 6E',
+    description: <Text size="xlarge">32</Text>,
+    icon: <Iteration height="medium" />,
+  },
+  {
+    value: 'storage management',
+    title: 'Aruba Mobility Controller Virtual',
+    description: <Text size="xlarge">6</Text>,
+    icon: <Iteration height="medium" />,
+  },
+];
+export const SupportSelector = () => {
+  return (
+    <Page>
+      <PageContent>
+        <Box gap="medium">
+          <Box gap="small">
+            <Heading margin="none">What can we help you with?</Heading>
+            <Text>Choose a product to continue.</Text>
+          </Box>
+          <TextInput
+            aria-label="search"
+            icon={<SearchIcon />}
+            placeholder="Search all assets"
+            type="search"
+            width="medium"
+          />
```

Comment:
Can we remove the search here since it is not functional?

Reply:
Is it valuable to have if someone were to copy + paste this template?